### PR TITLE
Created overlay for the Moto Z2 Play (albus).

### PR DESCRIPTION
**Pull request by deivesj**

### CODE REVIEW

**Comment by phhusson** [comment]
Remove fail file from the commit

File: Moto/Z2PLay/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.moto.z2play"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
```
Comment (phhusson) [inline Moto/Z2PLay/AndroidManifest.xml]: Check moto e5 overlay to see how to bypass this error

File: Moto/Z2PLay/res/values/config.xml [hunk 2]
```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<bool name="config_enableAutoPowerModes">true</bool>
+	<integer name="config_autoPowerModeAnyMotionSensor">30</integer>
+	<bool name="config_wifi_dual_band_support">true</bool>
+	<bool name="config_wifi_fast_bss_transition_enabled">true</bool>
+	<bool name="config_wifi_background_scan_support">true</bool>
+	<bool name="config_wifi_enable_disconnection_debounce">true</bool>
+	<bool name="config_setColorTransformAccelerated">true</bool>
+	<bool name="config_nightDisplayAvailable">false</bool>
```
Comment (phhusson) [inline Moto/Z2PLay/res/values/config.xml]: You're sure you want that?

File: Moto/Z2PLay/res/values/config.xml [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+	<bool name="config_dozeAfterScreenOff">true</bool>
+	<bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>
+	<bool name="config_cellBroadcastAppLinks">true</bool>
+	<integer name="config_mobile_mtu">1410</integer>
+	<bool name="config_speed_up_audio_on_mt_calls">true</bool>
+	<bool name="config_auto_attach_data_on_creation">false</bool>
+	   <string name="config_wifi_tcp_buffers">131072,262144,3145728,4096,221184,3145728</string>
+    <bool name="config_unplugTurnsOnScreen">true</bool>
+    <bool name="config_lidControlsSleep">false</bool>
+    <bool name="config_useDevInputEventForAudioJack">false</bool>
```
Comment (phhusson) [inline Moto/Z2PLay/res/values/config.xml]: This sounds wrong, Qualcomm device have that to true
Where did you get this value from?

File: Moto/Z2PLay/res/values/config.xml [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        <item>117</item>
+        <item>129</item>
+        <item>141</item>
+        <item>153</item>
+        <item>166</item>
+        <item>179</item>
+        <item>192</item>
+        <item>205</item>
+        <item>218</item>
+        <item>256</item>
```
Comment (phhusson) [inline Moto/Z2PLay/res/values/config.xml]: I'm pretty sure 256 doesn't work, it should be 255

**Comment by deivesj** [comment]
> What have you tested? What is fixed with this overlay? Why can't this overlay be included in vendor, since that's a community treble and not an OEM treble?

Ok, we forgot everything else, yes, I already have framework-res__auto_generated_rro.apk in an unofficial treble vendor, just thought of having its overlay structure with albus. Anyway thank you for your attention and your excellent service to the whole community.

File: Moto/Z2PLay/res/values/config.xml (hunk 4 above)
Comment (deivesj) [reply Moto/Z2PLay/res/values/config.xml]: fix, thanks.

**Comment by TadiT7** [comment]
Overlays for unofficial Treble devices are just unnecessary clutter, sorry

**Pull request closed without merging**